bin/node-template/node/src/service.rs

```diff
@@ -126,7 +126,7 @@ pub fn new_partial(
 						slot_duration,
 					);
 
-				Ok((timestamp, slot))
+				Ok((slot, timestamp))
```

Comment:
Here timestamp inherent is still required but not because of the slot-based algorithms.
It is required by the `pallet_timestamp` (as before)